cloud_governance/policy/helpers/aws/aws_policy_operations.py

```diff
@@ -132,10 +132,8 @@ def update_resource_day_count_tag(self, resource_id: str, cleanup_days: int, tag
             elif self._policy == 'empty_roles':
                 self._iam_client.tag_role(RoleName=resource_id, Tags=tags)
             elif self._policy in ('ip_unattached', 'unused_nat_gateway', 'zombie_snapshots', 'unattached_volume',
-                                  'instance_run'):
+                                  'instance_run', 'instance_idle'):
```

Comment:
Why did you delete instance_idle ?

Reply:
This is for tagging.

Reply:
If the policy is instance_idle, we need to add tags.

Reply:
The days count tag ?

Reply:
`tags = self._update_tag_value(tags=tags, tag_name='DaysCount', tag_value=str(cleanup_days))`
This code is above, it will take care of updation.